src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -175,28 +212,67 @@ private void createTableDescriptionTableIfNotExists(int pollIntervalSeconds) {
+            .withLocalSecondaryIndexes(getLocalIndexes(description.getLocalSecondaryIndexes()));
+    }
+
+
+    private List<GlobalSecondaryIndex> getGlobalIndexes(Collection<GlobalSecondaryIndexDescription> descriptions) {
+        return descriptions == null ?  null : descriptions
+            .stream()
+            .map(s->
+                new GlobalSecondaryIndex().withIndexName(s.getIndexName())
+                .withKeySchema(s.getKeySchema())
+                .withProjection(s.getProjection())
+                .withProvisionedThroughput(getProvisionedThroughput(s.getProvisionedThroughput())))
+            .collect(Collectors.toList());
+    }
+
+    private List<LocalSecondaryIndex> getLocalIndexes(Collection<LocalSecondaryIndexDescription> descriptions) {
+        return descriptions == null ? null :
+            descriptions.stream()
+                .map(s->
+                    new LocalSecondaryIndex()
+                        .withIndexName(s.getIndexName())
+                        .withKeySchema(s.getKeySchema())
+                        .withProjection(s.getProjection()))
+                .collect(Collectors.toList());
+    }
+
+    private ProvisionedThroughput getProvisionedThroughput(ProvisionedThroughputDescription description) {
+        return description == null ? null :
+            new ProvisionedThroughput(description.getReadCapacityUnits(), description.getWriteCapacityUnits());
+    }
+
     private Map<String, AttributeValue> createItem(CreateTableRequest createTableRequest) {
         TableDescription tableDescription = new TableDescription()
                 .withTableName(createTableRequest.getTableName())
                 .withKeySchema(createTableRequest.getKeySchema())
                 .withAttributeDefinitions(createTableRequest.getAttributeDefinitions())
                 .withStreamSpecification(createTableRequest.getStreamSpecification());
 
-        // Only set provisioned throughput if the BillingMode is not PAY_PER_REQUEST
-        DynamoDbCapacity capacityUtil = new DynamoDbCapacity();
-        final boolean setProvisionedThroughput = (createTableRequest.getBillingMode() == null
-                || !createTableRequest.getBillingMode().equals(BillingMode.PAY_PER_REQUEST.toString()));
-
-        if (setProvisionedThroughput) {
-            long readCapacityUnits = capacityUtil.getCapacity(
-                    createTableRequest.getProvisionedThroughput(), DynamoDbCapacity.CapacityType.READ);
-            long writeCapacityUnits = capacityUtil.getCapacity(
-                    createTableRequest.getProvisionedThroughput(), DynamoDbCapacity.CapacityType.WRITE);
-
-            tableDescription.withProvisionedThroughput(new ProvisionedThroughputDescription()
-                    .withReadCapacityUnits(readCapacityUnits)
-                    .withWriteCapacityUnits(writeCapacityUnits));
-        }
+        // // Only set provisioned throughput if the BillingMode is not PAY_PER_REQUEST
```

Comment:
I commented this provisioned throughput business out to get feedback from y'all, but I fundamentally disagree with where these defaults are being applied. 

IMO, we should take the exact CreateTableRequest the customer is giving us, and storing that in our metadata table. If we're operating against a shared table mt-dynamo, or we're operating against a pay per request table, we should ignore any provisioned throughput the customer specifies, but if we set defaults here, and migrate a tenant from a pay-per-request instance over to a table-per-tenant instance without pay-per-request for example, we lose any configuration the customer configured, same story for vice versa.